packages/flutter_localizations/test/translations_test.dart

```diff
@@ -0,0 +1,89 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_localizations/flutter_localizations.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  testWidgets('translations exist for all supported languages', (WidgetTester tester) async {
+    final List<String> languages = <String>[
+      'ar', // Arabic
+      'de', // German
+      'en', // English
+      'es', // Spanish
+      'fa', // Farsi (Persian)
+      'fr', // French
+      'he', // Hebrew
+      'it', // Italian
+      'ja', // Japanese
+      'ps', // Pashto
+      'pt', // Portugese
+      'ru', // Russian
+      'sd', // Sindhi
+      'ur', // Urdu
+      'zh', // Chinese (simplified)
+    ];
+
+    for (String language in languages) {
```

Comment:
You could move this loop outside `testWidgets` and include the language in the test description. When the test fails the error message will contain the locale name, for which the test failed. E.g.:

```dart
for (var lang in languages) {
  testWidgets('translations exist for $lang', () {
    ...
  });
}
```

Reply:
Good idea, I did that.